src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,796 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { act, fireEvent, render } from '@testing-library/react';
+import React from 'react';
+import { Slider, focusThumb } from './Slider';
+
+const createBoundingClientRect = (
+  left: number,
+  top: number,
+  right: number,
+  bottom: number,
+): DOMRect => ({
+  left,
+  top,
+  right,
+  bottom,
+  width: right - left,
+  height: bottom - top,
+  x: left,
+  y: top,
+  toJSON: () => '',
+});
+
+/**
+ * Setup default size for slider container to be used by all slider tests
+ */
+const getBoundingClientRect = Element.prototype.getBoundingClientRect;
+const sliderContainerSize = createBoundingClientRect(10, 0, 1010, 60);
+Element.prototype.getBoundingClientRect = () => sliderContainerSize;
+
+afterAll(() => {
+  Element.prototype.getBoundingClientRect = getBoundingClientRect;
+});
+
+const defaultSingleValue = [50];
+
+const assertBaseElement = (container: HTMLElement) => {
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+};
+
+it('should render correctly in its most basic state', () => {
+  const { container } = render(<Slider values={defaultSingleValue} />);
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-disabled')).toEqual('false');
+});
```

Comment:
Can you add new line between all test cases? Hard to read when the whole file looks like one long test

Reply:
I had spaces between the lines and something remove them, I thought maybe you guys had a commit hook that did that. IO will put the spaces back in.

Reply:
Spaces are back in... until the gremlins eat them again,